modules/dnn/src/layers/fast_convolution/fast_convolution_internals.hpp

```diff
@@ -0,0 +1,25 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#ifndef OPENCV_FAST_CONVOLUTION_INTELNALS_HPP
+#define OPENCV_FAST_CONVOLUTION_INTELNALS_HPP
+
+#include "fast_convolution.hpp"
+#include "opencv2/core/hal/hal.hpp"
+#include "opencv2/core/hal/intrin.hpp"
+#include "../layers_common.hpp"
+
+namespace cv { namespace dnn {
+
+void runDepthwise(InputArray _input, OutputArray _output, const Ptr<FastConv2d>& conv, float minval, float maxval,
```

Comment:
Why not put these declarations to fast_convolution.hpp? Why do we need a separate tiny header?

Reply:
fixed.